Translate PULL_REQUEST_TEMPLATE.md to english (#113)
adessoAG/devblog: PULL_REQUEST_TEMPLATE.md

> If you make a pull request, please let us know your changes by using the publish checklist below. 

Reviewer: ```suggestion
If you create a pull request, please log your changes by using the publish checklist below. 
```

# Publish Checklist

- **Meta** (to be ticked off by the author)

>   - [ ] [Item file stored in the right place](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#dateiname-und-ablageort)

Reviewer: ```suggestion
  - [ ] [Article text file stored in the correct place](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#dateiname-und-ablageort)
```

  - [ ] [Author info maintained](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#autoren-informationen)
  - [ ] [Metadata maintained:](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#metadaten)
    - [ ] "title" assigned
    - [ ] "author" specified

>     - [ ] "categories" contains at most one of the main categories (Java, Microsoft, Methodology, Architectures, Software Development, Business & People)

Reviewer: ```suggestion
    - [ ] "categories" contains at most one of the main categories (Java, Microsoft, Methodik, Architekturen, Softwareentwicklung, Business & People)
```

Du kannst die Kategorien hier nicht einfach umbenennen, sonst kommt der Blog damit nicht klar.

    - [ ] "tags" contains (any) matching tags
- **Text** (to be ticked off by the author)
  - [ ] [Teaser complies with specifications](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#einleitung--teaser)
  - [ ] Spelling corrected

>   - [ ] [Indent to one sentence per line](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#ein-satz-pro-zeile)

Reviewer: ```suggestion
  - [ ] [Each sentence is on its own line](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#ein-satz-pro-zeile)
```

  - [ ] [Images are correctly included in the article](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#bilder)
  - [ ] [Headings correspond to the specifications](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#%C3%9Cberschriften)
  - [ ] Code snippets have been provided with [Syntax Highlighting](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#syntax-highlighting)
  - [ ] The article was reviewed in the [Preview](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#online-preview) 
- **Review** (to be ticked off by the reviewer)
  - [ ] Spelling corrected
  - [ ] The article was reviewed in the [Preview](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#online-preview) 

>   - [ ] CCO was informed about new article

Reviewer: ```suggestion
  - [ ] CCO was informed about the new article
```

  - [ ] Date specifications 
    - [ ] match in header and file name

>     - [ ] stand on today (but in the past)

Reviewer: ```suggestion
    - [ ] are set to today (but in the past)
```

Author: Done.
